articles/azure-functions/durable/durable-functions-troubleshooting-guide.md

```diff
@@ -129,7 +129,7 @@ traces
 | extend status = customDimensions["prop__status"]
 | extend details = customDimensions["prop__Details"] 
 | extend reason = customDimensions["prop__reason"]
-| where severityLevel > 1 // to see all logs of  severity level "Information" or greater.
```

Comment:
Thanks for the PR. Shouldn't this to be >= 1 , not =>1?

Reply:
@jaliyaudagedara Could you review the comment above?

Reply:
duplicate comment. Resolving